single-audit/manage.py

```diff
@@ -3,6 +3,7 @@
 import sys
 
 if __name__ == '__main__':
+    os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'single_audit.settings')
```

Comment:
@fureigh Could you say more about what this line is doing?

Reply:
@alexsoble I tried to explain here: https://github.com/18F/federal-grant-reporting/pull/99/commits/2224a5d337ee42162ec726f93f4ef86a5c27f433 If you have follow-up questions or concerns, lmk.